Flepper.QueryBuilder/Commands/SelectCommand.cs

```diff
@@ -8,7 +11,11 @@ internal class SelectCommand : BaseQueryBuilder, ISelectCommand
         public SelectCommand() 
             => Command.Append("SELECT * ");
 
-        public SelectCommand(params string[] columns) 
-            => Command.AppendFormat("SELECT {0}", columns.JoinColumns());
+       public SelectCommand(params SqlColumn[] columns)
+        {
+            if (columns.Any(c => c == null))
+                throw new ArgumentNullException(nameof(columns), "All columns names should not be null");
+            Command.AppendFormat("SELECT {0}", columns.Select(c => c?.ToString()).JoinColumns());
```

Comment:
You can remove the null propagation operator `?.` since the previous if protect against null